executing/executing.py

```diff
@@ -530,6 +539,110 @@ def visit_ClassDef(self, node):
         self.stack.pop()
 
 
+class PositionNodeFinder(object):
+    def __init__(self, frame, stmts, tree, lasti, source):
+        # we can use co_positions() since 3.11, which has fewer limitations
+
+        bc_list = list(dis.Bytecode(frame.f_code, show_caches=True))
+
+        def find_node(index, ignore_positions=(), typ=None):
+            position = bc_list[index // 2].positions
```

Comment:
I think the name `positions` would make a bit more sense to match the attribute name.